genpass/database.py

```diff
@@ -0,0 +1,35 @@
+import sqlite3
+
+
+class DatabaseConnection():
+	def __init__(self):
+		# Connecting to sqlite3 database
+		self.con = sqlite3.connect('generated_password.db')
+		self.cursor_obj = self.con.cursor()
+	def create_table(self):
+		# Creating table
+		self.cursor_obj.execute("CREATE TABLE passwords(id integer PRIMARY KEY, portal_name text, portal_url real, user_email text, tag text, creation_date text, last_modified text, notes text, level text, importance text)")
+		self.con.commit()
```

Comment:
For user_email field, the data type should be ```varchar``` as it contains special chars and numbers too. what do you think?

Reply:
It's ok to use ``varchar`` but we can also use ``nvarchar`` which uses UNICODE which will help us to get multiple unique codes.
I think it will be beneficial to store multiple emails.
``nvarchar``(contains 320 char)=64 char+@+255 domain name.
``varchar``(contains 255 char)!
What do you think?